### PR TITLE
fixing documentation, issue 22697

### PR DESCRIPTION
**Pull request by mingbowan**
As fmassa commented :

> Agree, it should probably be weight, start, end

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -2449,7 +2449,7 @@ def merge_dicts(*dicts):
 
 The shapes of :attr:`start` and :attr:`end` must be
 :ref:`broadcastable <broadcasting-semantics>`. If :attr:`weight` is a tensor, then
-the shapes of :attr:`start`, :attr:`end` must be :ref:`broadcastable <broadcasting-semantics>`.
+:attr:`weight`, :attr:`start`, and :attr:`end` must be :ref:`broadcastable <broadcasting-semantics>`.
```
Comment (vishwakftw) [inline torch/_torch_docs.py]: Why was “the shapes of “ removed?

Comment (mingbowan) [reply torch/_torch_docs.py]: Must've been a mistake. Added back.

**Review by vishwakftw — APPROVED** [review]
Looks good to me, thank you for fixing this!

**Pull request closed without merging**